hudi-common/src/main/java/org/apache/hudi/common/table/TableSchemaResolver.java

```diff
@@ -114,7 +113,7 @@ private MessageType getTableParquetSchemaFromDataFile() throws Exception {
           if (lastDeltaCommit.isPresent()) {
             HoodieInstant lastDeltaInstant = lastDeltaCommit.get();
             // read from the log file wrote
-            commitMetadata = HoodieCommitMetadata.fromBytes(activeTimeline.getInstantDetails(lastDeltaInstant).get(),
+            HoodieCommitMetadata commitMetadata = HoodieCommitMetadata.fromBytes(activeTimeline.getInstantDetails(lastDeltaInstant).get(),
```

Comment:
there is a chance that last delta commit is empty and does not have valid schema. so, the fix we have done for COW above, might also be required here.

Reply:
have optimized the mor part, please help to review this again.